README.md

```diff
@@ -1,7 +1,7 @@
 ![Mumble screenshot](screenshots/Mumble.png)
 
-Mumble - A voice chat utility for gamers
-=======================================
+# Mumble - OpenSource voice-chat software 
```

Comment:
Isn't Open Source written with a space? At least the [Open Source Initiative](https://opensource.org/) does that :thinking: 

Edit: dang I wanted to leave a single commit, not start a review :sweat_smile:

Reply:
Hm... I always write it as `OpenSource`. I think `Open Source` is pretty ugly, but the internet also uses `Open-Source` a lot which I think is much better :thinking:

Reply:
It's not really relevant anyway. I just haven't seen `OpenSource` yet. Kinda makes it look like a stylized brand name. But everyone still understands it sooo :shrug:

Reply:
We talked about this in our team meeting last week. Open Source seems to be the most common and our agreed upon writing style. We also use that on our website and repository description.

Reply:
Done (finally ^^)